datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/InstantDeserializer.java

```diff
@@ -519,14 +519,42 @@ private static String replaceZeroOffsetAsZ(String text)
     }
 
     // @since 2.13
-    private String addInColonToOffsetIfMissing(String text)
+    private static String addInColonToOffsetIfMissing(String text)
```

Comment:
I don't see any usage out of this class? May not need to be static

Reply:
This was partly drive by cleanup to indicate that the method does not touch any of the deserializer instance state -- it is a pure function that returns a `String` with a colon in the offset (and possibly returns the original input `text` string if the offset does not need a colon).